### DISCUSSION TITLE
Unable to completely send SOL from sollet to metamask using wormhole bridge

### DISCUSSION THREAD

**Discussion by debbiewdm** [wormhole-foundation/wormhole#521] [Problems / bugs] [answered]
Hello, I was able to send SOL tokens from my sollet wallet to metamask using wormhole bridge. When I get to the 'redeem tokens' section, I would approve the transaction in my metamask but it will take ages to go through and it never did go through. Please help. Thanks

**Comment by leoluk** [comment, marked as answer]
Please provide more details (tx hash, screenshots).

**Comment by johndkim722** [comment]
> 

Hi, a friend went through these steps to send SOL to his metamask wallet, but we can't find his SOL anywhere. How can we recover his SOL or see his SOL?
